supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -152,12 +152,12 @@ The practice of having both modes together is often referred to as _hybrid_, _ag
 
 [discrete]
 [[bimonthly]]
-==== image:images/yes.png[yes] bimonthly (adverb)
-*Description*: _Bimonthly_ means every other month.
+==== image:images/no.png[no] bimonthly (adverb)
+*Description*: Do not use. _Bimonthly_ can mean either twice a month or every two months. Instead, write "twice a month" or "every two months" to remove ambiguity.
 
-*Use it*: yes
+*Use it*: no
 
-*Incorrect forms*: bi-monthly
+*Incorrect forms*:
 
 *See also*:
```

Comment:
"See also: biweekly" and visa versa?

Establish a close relationship between the two terms?